test/test.transporters.js

```diff
@@ -79,7 +90,9 @@ describe('Transporters', function() {
     var urlshortener = google.urlshortener('v1');
     var obj = { longUrl: 'http://google.com/' };
     urlshortener.url.insert({ resource: obj }, function(err, result) {
-      assert.deepEqual(err, { code: 500, message: 'There was an error!' });
+      assert(err instanceof Error);
+      assert.equal(err.code, 500);
+      assert.equal(err.message, 'There was an error!');
```

Comment:
That's fair. I think as of 0.10.x they put the stack object in the errors by default instead of having to capture it with captureStackTrace. Considering we only support 0.10.x this is a fair assumption.